packages/commonwealth/client/scripts/views/components/PreferenceTags/usePreferenceTags.tsx

```diff
@@ -1,27 +1,41 @@
-import { useState } from 'react';
-import { preferenceTags } from './mockedTags';
+import { useFetchTagsQuery } from 'client/scripts/state/api/tags';
+import { useEffect, useRef, useState } from 'react';
 import { PreferenceTagsHookProps, SelectedTag, Tag } from './types';
 
 const usePreferenceTags = ({
   initialSelectedTag = [],
```

Comment:
I can't see anywhere the usage of `initialSelectedTag` param

Reply:
It was used in earlier variations, but it's not needed now. I do however see future use cases where we might want to use this in other places like when we add more use cases for community/profile tags. 

Lmk if you want me to remove this.

Reply:
let's remove it for now, we can always add it when it is really needed